packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -161,64 +173,98 @@ const FilterConfiguration: FC<Props> = ({
     );
 
     return (
-        <ConfigureFilterWrapper>
-            <Group spacing="xs">
-                <FieldIcon item={field} />
-                <FieldLabel item={field} />
-            </Group>
-
+        <Stack>
             <Tabs
                 value={selectedTabId}
-                onTabChange={(tabId: FilterTabs) => onTabChange(tabId)}
+                onTabChange={(tabId: FilterTabs) => setSelectedTabId(tabId)}
             >
                 {isCreatingNew || isEditMode || isTemporary ? (
                     <Tabs.List mb="md">
                         <Tooltip
                             label="Select the value you want to filter your dimension by"
                             position="top-start"
                         >
-                            <Tabs.Tab value="settings">Settings</Tabs.Tab>
+                            <Tabs.Tab value={FilterTabs.SETTINGS}>
+                                Settings
+                            </Tabs.Tab>
                         </Tooltip>
 
                         <Tooltip
                             label="Select tiles to apply filter to and which field to filter by"
                             position="top-start"
                         >
-                            <Tabs.Tab value="tiles">Tiles</Tabs.Tab>
+                            <Tabs.Tab
+                                value={FilterTabs.TILES}
+                                disabled={!field}
+                            >
+                                Tiles
+                            </Tabs.Tab>
                         </Tooltip>
                     </Tabs.List>
                 ) : null}
 
-                <Tabs.Panel value="settings">
-                    <FilterSettings
-                        isEditMode={isEditMode}
-                        field={field}
-                        filterRule={internalFilterRule}
-                        onChangeFilterOperator={handleChangeFilterOperator}
-                        onChangeFilterRule={handleChangeFilterRule}
-                        popoverProps={popoverProps}
-                    />
-                </Tabs.Panel>
+                <Tabs.Panel value={FilterTabs.SETTINGS} w={350}>
+                    <Stack>
+                        {!!fields && isCreatingNew ? (
+                            <FormGroup
+                                style={{ marginBottom: '5px' }}
+                                label={
+                                    <Text>Select a dimension to filter</Text>
```

Comment:
from the designs i would add: 
```suggestion
                                    <Text fw={500} >Select a dimension to filter</Text>
```

and a start to indicate it's required... perhaps `useForm` from `mantine`? 🤔

Reply:
I will add the fw and star, but not bring in useForm for now. There's a lot to that. The mantine input has 'required' 😢

Reply:
sounds good!